### PR TITLE
Add codeowners for bundler and wrapper tools

### PR DESCRIPTION
**Pull request by Fznamznon**
Since these tools are placed in clang/ directory, the automation
requests review from clang frontend codeowners, this is not correct. Add
proper codeowners for these tools.

### CODE REVIEW

**Comment by bader** [comment]
@Fznamznon, please, resolve merge conflicts.

@kbobrovs, @sndmitriev, please, confirm that you okay to be code owners for these tools.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -88,3 +88,6 @@ xpti/ @tovinkere @andykaylor
 xptifw/ @tovinkere  @andykaylor
 
 llvm/tools/sycl-post-link/ @kbobrovs @AlexeySachkov
+
+clang/tools/clang-offload-bundler/ @kbobrovs @sndmitriev
+clang/tools/clang-offload-wrapper/ @kbobrovs @sndmitriev
```
Comment (kbobrovs) [inline .github/CODEOWNERS:93]:
```suggestion
clang/tools/clang-offload-wrapper/ @sndmitriev @kbobrovs
```
as @sndmitriev is the author of the tool

Comment (Fznamznon) [reply .github/CODEOWNERS]: Sure, I didn't know that.

**Review by kbobrovs — APPROVED** [review]

**Review by bader — APPROVED** [review]
@sndmitriev, please, confirm that your are okay with that change.

**Review by sndmitriev — APPROVED** [review]

**Pull request merged**